dockerfiles/stardog.cache.properties

```diff
@@ -0,0 +1,16 @@
+# # Flag to run this server as a cache target
```

Comment:
It would be better if we removed anything that is not relevant to cached nodes instead of docing them out.  In the properties the only things doced out should be things that you might undo to change the defaults.

Reply:
Actually they are all relevant.
They are commented out because currently we don't really need to specify a cache target in the properties. it only helps when creating cached virtual graphs, in which case we don't need to register the virtual graph within the cache target as well. if we configure the cache target with the properties file, we could point it to the same zookeeper as the main instance, and the cache could read the VG configs from zk. I think this is the preferred method (using stardog properties) and we could enable it future iterations.